wf/en-us/Content/Topics/Workflows/function-reference/DateTime/datetime_diff.htm

```diff
@@ -1,4 +1,14 @@
 <!DOCTYPE html>
+<<<<<<< HEAD
```

Comment:
This is not right, looks like we didn't propertly fix a merge conflict
This comment applies to all files in `wf/en-us/Content/Topics/Workflows/function-reference/DateTime` folder

Reply:
so sorry, that's not good.
i'll create a separate PR, and do a repo-wide scrub for any of these that might be hanging around.

Reply:
https://github.com/okta/okta-help/pull/927